sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client.hpp

```diff
@@ -0,0 +1,324 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "azure/attestation/attestation_client_models.hpp"
+#include "azure/attestation/attestation_client_options.hpp"
+#include <azure/core/context.hpp>
+#include <azure/core/url.hpp>
+#include <string>
+
+namespace Azure { namespace Core { namespace Http { namespace _internal {
+  class HttpPipeline;
+}}}} // namespace Azure::Core::Http::_internal
+
+namespace Azure { namespace Security { namespace Attestation {
+
+  /** An AttestationToken represents an RFC 7519 JSON Web Token returned from the attestation
+   * service with the specialized body type.
+   * <typeparam name="T"></typeparam> The type which represents the body of the attestation token.
+   */
+  template <typename T> class AttestationToken final {
+  public:
+    /**
+     * @brief The full RFC 7515 JWS/JWT token returned by the attestation service.
+     */
+    std::string RawToken;
+
+    /**
+     * @brief      The elements of the raw token which will be signed by the Signature.
```

Comment:
nit: spacing inconsistency (here/elsewhere), something to clean up across the board

```suggestion
     * @brief The elements of the raw token which will be signed by the Signature.
```